h/static/scripts/directive/about-this-version-dialog.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name aboutThisVersionDialog
+ * @description Displays product version and environment info
+ */
+// @ngInject
+module.exports = function () {
+  return {
+    bindToController: true,
+    controllerAs: 'vm',
+    // @ngInject
+    controller: function ($scope, $window, settings, crossframe) {
+      this.userAgent = $window.navigator.userAgent;
+      this.version = settings.release;
+      this.dateTime = new Date();
```

Comment:
@robertknight I think it's actually quite useful to have the date.  I added a note on the trello card advocating for it last week.  https://trello.com/c/LGxxgOwd/298-add-product-version-and-environment-info-to-the-tool

TL:DR, there are all kinds of reasons why knowing exactly when someone reported an issue with our system might be important. For instance, their issue may have been the result of an operational situation rather than a code issue.  Without the date we don't know exactly when the issue was, and whether perhaps it might not even be a current problem.  It seems a simple thing to include.